### PR TITLE
Printchard/pbac

### PR DESCRIPTION
**Pull request by lCDSLl-Richard**

### CODE REVIEW

File: src/controllers/actionable.controller.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const renderActionables = async (req, res, next) => {
     const state = req.params.state;
     if (["pending", "accepted", "rejected"].includes(state)) {
       const actionables = await Actionable.getAllByState(state);
+      console.log(actionables);
```
Comment (DanielSebasCM) [inline src/controllers/actionable.controller.js:12]:
```suggestion
```

File: src/middlewares/privilege.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const User = require("../models/user.model");
+
+const authorize = (privileges) => {
+  return async (req, res, next) => {
+    if (!req.session.currentUser) {
+      return res.redirect("/login");
+    }
+
+    const builtUser = new User(req.session.currentUser);
+    let userPrivileges = await builtUser.getPrivileges();
```
Comment (DanielSebasCM) [inline src/middlewares/privilege.js:10]: You dont need to build a new user, you can get the db one
```suggestion
    const user =  User.getById(req.session.currentUser.uid);
    let userPrivileges = await user.getPrivileges();
```

File: src/routes/actionable.routes.js
Comment (DanielSebasCM) [inline src/routes/actionable.routes.js:1]: controller should be merged on auth?

**Review by DanielSebasCM — COMMENTED** [review]
I added some comments for others to review

**Review by sgarnica1 — APPROVED** [review]

**Pull request merged**